tests/filters/test_value_structured_token_check.py

```diff
@@ -3,33 +3,35 @@
 
 import pytest
 
+from credsweeper.common.constants import LATIN_1
 from credsweeper.filters import ValueStructuredTokenCheck
 from tests.filters.conftest import LINE_VALUE_PATTERN
 from tests.test_utils.dummy_line_data import get_line_data
 
 
 class TestValueStructuredTokenCheck:
 
-    @pytest.mark.parametrize("line", ["12345:asbdsa:28yd"])
+    @pytest.mark.parametrize("line", ["12345:«VÏel\tSpÁß¡»q¾ä×^0÷ÕJür°eÆ\r"])
     def test_value_structured_token_check_p(self, file_path: pytest.fixture, line: str) -> None:
```

Comment:
Why this test is marked _p and following _n? It seems that their conditions are inverted

Reply:
Order P, N - is a requirement.
Filter returns true if the value must be filtered. False - when it should be kept (positive value for rule).